src/pages/en/core-concepts/astro-components.md

```diff
@@ -175,6 +163,34 @@ const visible = true;
 {visible ? <p>Show me!<p> : <p>Else show me!</p>}
 ```
 
+### Dynamic Tags
+
+Local variables can also represent dynamic HTML _tags_:
+
+```astro title="src/components/DynamicTags.astro" /Tag|<Tag>|</Tag>/
+---
+import MySpecialHeader from "../MySpecialHeader.astro"
```

Comment:
```suggestion
```
There doesn't seem to be a reason to include this import here, as it's never used in the example? But I do like the rest of the example!

Reply:
Totally was supposed to say `Four different headers` and that first example had a import that wasn't supposed to be there 🙈. And I've got a less forced example up my sleeve!